tools/generate_conda_file.py

```diff
@@ -46,7 +46,7 @@
     "mock": "mock==2.0.0",
     "nltk": "nltk>=3.4",
     "numpy": "numpy>=1.13.3",
-    "pandas": "pandas>=0.23.4,<1.0.0",
```

Comment:
we had problems in the past with pandas >1 https://github.com/microsoft/recommenders/pull/1052. There are multiple versions from 0.23.4 to <1 https://pypi.org/project/pandas/#history, is there an easy easy way to know which of them are ok?

Reply:
CG says we need to be above 1.0.3

Reply:
can we set to >1.0.3,<2.0

Reply:
Ill should have added upper bound with the latest release. We don't want to up this pin until the latest version is released and verified... 
im not worrying about this for the rest of the packages in this PR, at some point I will.